cmd/status/order/order.go

```diff
@@ -0,0 +1,304 @@
+package status_order
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/quickfixgo/tag"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+	"sylr.dev/fix/pkg/cli/options"
+
+	"github.com/quickfixgo/enum"
+	"github.com/quickfixgo/field"
+	"github.com/quickfixgo/fixt11"
+	"github.com/quickfixgo/quickfix"
+
+	"sylr.dev/fix/config"
+	"sylr.dev/fix/pkg/cli/complete"
+	"sylr.dev/fix/pkg/dict"
+	"sylr.dev/fix/pkg/errors"
+	"sylr.dev/fix/pkg/initiator"
+	"sylr.dev/fix/pkg/initiator/application"
+	"sylr.dev/fix/pkg/utils"
```

Comment:
```suggestion
	"github.com/google/uuid"
	"github.com/quickfixgo/tag"
	"github.com/rs/zerolog"
	"github.com/spf13/cobra"

	"github.com/quickfixgo/enum"
	"github.com/quickfixgo/field"
	"github.com/quickfixgo/fixt11"
	"github.com/quickfixgo/quickfix"

	"sylr.dev/fix/config"
	"sylr.dev/fix/pkg/cli/complete"
	"sylr.dev/fix/pkg/cli/options"
	"sylr.dev/fix/pkg/dict"
	"sylr.dev/fix/pkg/errors"
	"sylr.dev/fix/pkg/initiator"
	"sylr.dev/fix/pkg/initiator/application"
	"sylr.dev/fix/pkg/utils"
```

Reply:
I've merged the 2 external import blocks